crates/turborepo-lib/src/config/user.rs

```diff
@@ -92,7 +92,8 @@ impl UserConfigLoader {
 
         let config = Config::builder()
             .add_source(raw_disk_config.clone())
-            .add_source(Environment::with_prefix("turbo").source(environment))
+            .add_source(Environment::with_prefix("TURBO").source(environment.clone()))
+            .add_source(Environment::with_prefix("VERCEL_ARTIFACTS").source(environment))
```

Comment:
Not important, but these prefixes all get made lowercase: https://github.com/mehcode/config-rs/blob/master/src/env.rs#L247

Reply:
yeah I mostly made them uppercase because I tend to see environment variables in uppercase and it was confusing me to see the prefixes as lowercase.